workers/analyzer/src/test/kotlin/UtilsTest.kt

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright (C) 2025 The ORT Server Authors (See <https://github.com/eclipse-apoapsis/ort-server/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.eclipse.apoapsis.ortserver.workers.analyzer
+
+import io.kotest.core.spec.style.WordSpec
+import io.kotest.matchers.nulls.shouldNotBeNull
+import io.kotest.matchers.shouldBe
+
+import org.eclipse.apoapsis.ortserver.model.runs.Identifier
+
+import org.ossreviewtoolkit.model.Identifier as OrtIdentifier
+import org.ossreviewtoolkit.model.Project as OrtProject
+import org.ossreviewtoolkit.model.VcsInfo as OrtVcsInfo
+import org.ossreviewtoolkit.model.VcsType
+
+class UtilsTest : WordSpec({
+    "getIdentifierToShortestPathMap" should {
```

Comment:
Nit: We tend to use "()" after the function name to denote it's a function that's being tested.